app/views/data_sources/show.haml

```diff
@@ -1,29 +1,27 @@
 = render 'breadcrumbs'
 %h1= @data_source.name
-= simple_form_for @data_source, url: data_source_path(@data_source) do |f|
-  .row
-    .col-sm-8
-      %table.table.table-condensed
-        %tr
-          %th Last Import Run
-          %td= @data_source.last_imported_at
-        %tr
-          %th Date Span
-          %td
-            - if @data_source.data_span.present?
-              = @data_source.data_span.join(' - ')
-        %tr
-          %th
-            Visible in the Window into the Warehouse?
-          %td
-            = f.input :visible_in_window, label: false, input_html: {name: 'data_source[visible_in_window]'}
-    .col-sm-4
-      = link_to new_data_source_upload_path(@data_source), class: 'btn btn-primary' do
-        %span.icon-plus
-        Upload HUD HMIS ZIP
-  - if @data_source.has_data?
-    .row.table-utilities
-      .col-sm-12
+.row
+  .col-sm-8
+    %table.table.table-condensed.table-hover
+      %tr
+        %th Last Import Run
+        %td= @data_source.last_imported_at
+      %tr
+        %th Date Span
+        %td
+          - if @data_source.data_span.present?
+            = @data_source.data_span.join(' - ')
+  .col-sm-4
+    = link_to new_data_source_upload_path(@data_source), class: 'btn btn-primary' do
+      %span.icon-plus
+      Upload HUD HMIS ZIP
+= link_to "/data_sources/#{@data_source.id}/uploads", class: 'btn btn-primary' do
+  Imports
+- if @data_source.has_data?
+  .row.table-utilities
+    .col-sm-12
+      = simple_form_for @data_source, url: data_source_path(@data_source) do |f|
```

Comment:
You'll need to merge release-10 first.  This breaks what was there. (The visible in warehouse should be up in the table above.